pkg/database/mariadb.go

```diff
@@ -9,15 +9,30 @@ import (
 
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/huandu/go-sqlbuilder"
-	"github.com/jmoiron/sqlx"
 	"github.com/kubearchive/kubearchive/pkg/database/facade"
 )
 
 func init() {
-	RegisteredDatabases["mariadb"] = NewMariaDBDatabase
+	RegisteredDatabases["mariadb"] = NewMariaDBDatabase()
 	RegisteredDBCreators["mariadb"] = NewMariaDBCreator
 }
 
+type MariaDBDatabase struct {
```

Comment:
same as previous comment

```suggestion
type mariaDBDatabase struct {
```

Reply:
Applied!